src/text_file_functions.cpp

```diff
@@ -449,9 +450,10 @@ QColor AOApplication::get_chat_color(QString p_identifier, QString p_chat)
 {
   QColor return_color(255, 255, 255);
 
-  QString design_ini_path = get_base_path() + "misc/" + p_chat + "/config.ini";
+  QString design_ini_path =
+      get_base_path() + "misc/" + get_chat(p_chat) + "/config.ini";
   QString default_path = get_base_path() + "misc/default/config.ini";
-  QString f_result = read_design_ini("c" + p_identifier, design_ini_path);
+  QString f_result = read_design_ini(p_identifier, design_ini_path);
```

Comment:
Wait losing the "c" doesn't break anything does it, was this tested?

Reply:
Is it because get_chat_color(p_identifier already supplies the "c" prefix?

Reply:
i don't remember making this change, i think clang-format decided to put it in

Reply:
Ok, can confirm, every call to the function adds the "c" in the parameters already, so no need to do it in the function itself.